src/logic/reminders.ts

```diff
@@ -0,0 +1,82 @@
+import { isTest } from '../utils/env';
+
+const REMINDER_ENDPOINT_PROD = 'https://support.theguardian.com/reminders/create/one-off';
+// const REMINDER_ENDPOINT_CODE = 'https://support.code.dev-theguardian.com/reminders/create/one-off';
+
+export type ReminderStatus = 'DEFAULT' | 'IN_PROGRESS' | 'SUCCESS' | 'FAILURE';
```

Comment:
Given that this is a UI state concern, I wonder if it'd be better to define this inline in the component? I don't _think_ anything else would need to access this type. What do you think?

Reply:
It's used by both the reminder button, and the newsletter subscribe button (from which I stole it)

Reply:
Oh are you sure it's being used by the newsletter subscribe button? I can't see that import in this PR. I don't feel super strongly, but I'd say on balance it's probably not worth the slight additional complexity to share this between the two types of buttons (newsletter signup and reminders) and would opt for duplicating. If we do want to share as you say renaming would be good (and maybe defining it in a file outside of this reminders specific code?)

Reply:
Ah ... I pushed a renaming fix, and moved to a types file, before reading this comment.